docs/components/DocSearch/wrapper.tsx

```diff
@@ -48,7 +48,7 @@ export default function () {
   }, [])
 
   return (
-    <div className="relative">
+    <div className="relative searchbar">
```

Comment:
This is only supposed to hide in mobile? Maybe a tailwind class like this would cover it then: 

``` 
max-md: hidden 
```

Reply:
agree

Reply:
Applied :+1: